doc/user/content/manage/reverse-etl/census.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Census"
+description: "How to use Census to export data out of Materialize."
+menu:
+  main:
+    parent: "reverse-etl"
+---
+
+This guide walks you through the steps required to create a [Census](https://www.getcensus.com/) sync using Materialize.
+
+## Before you begin
+
+In order to build a sync with Census you will need:
+
+* A table, view, materialized view or source within your Materialize account that you would like to export.
+* The connection details for one of [destinations](https://www.getcensus.com/integrations) supported by Census.
+
+## Step 1. Set up a Materialize Source
+
+To begin you will need to add your Materialize database as a source in Census.
+
+1. In Census navigate to **Sources** and select **New Source**.
```

Comment:
Looking at the other guides was unsure of the formatting to use for UI headers/sections. Currently using bold.

Reply:
I use the [Google Style Guide for UI Elements](https://developers.google.com/style/ui-elements). I think it is a safe and concise framework to follow.